sdk/tests/wallet/chrysalis_migration.rs

```diff
@@ -348,6 +348,20 @@ async fn migrate_chrysalis_stronghold() -> Result<()> {
     tear_down(storage_path)
 }
 
+#[tokio::test]
+async fn migrate_empty_chrysalis_db() -> Result<()> {
+    iota_stronghold::engine::snapshot::try_set_encrypt_work_factor(0).unwrap();
+    let storage_path = "migrate_empty_chrysalis_db";
```

Comment:
I feel like the test would be more relevant if we were actually trying to migrate a stardust db, surely we have some in fixtures we could use.

Reply:
Done